src/services/sync/strategies/TSP/TSPSchoolSyncer.js

```diff
@@ -249,15 +277,19 @@ class TSPSchoolSyncer extends mix(Syncer).with(ClassImporter) {
 		try {
 			const sourceOptions = {};
 			sourceOptions[SOURCE_ID_ATTRIBUTE] = tspTeacher.lehrerUid;
-			const teacher = await this.createUserAndAccount({
-				namePrefix: tspTeacher.lehrerTitel,
-				firstName: tspTeacher.lehrerVorname,
-				lastName: tspTeacher.lehrerNachname,
-				schoolId: school._id,
-				source: ENTITY_SOURCE,
-				sourceOptions,
-			},
-			'teacher', systemId);
+			const teacher = await createUserAndAccount(
```

Comment:
Is it is a performant way to create  users with >await< createUserAndAccount? I do not know the complet workflow, is it start as batch?

Reply:
Sadly this is how we do it and I don't think it can be changed:
1. create the user
2. create a registration pin
3. create and activate the account
4. return the user

The parallelization is happening one layer up: [line 136](https://github.com/schul-cloud/schulcloud-server/pull/1006/files#diff-c9b8941d97d31db3160022c1b5788fdbR136)